packages/bruno-app/src/utils/importers/postman-environment.js

```diff
@@ -31,8 +31,9 @@ const importPostmanEnvironmentVariables = (brunoEnvironment, values) => {
 };
 
 const importPostmanEnvironment = (environment) => {
+  const sanitizedEnvironmentName = environment.name.replace(/\//g, '_');
```

Comment:
This will not work for `Some / Environment / Name` as it'll result in `Some _ Environment / Name`, also there are more forbidden characters. 

Any character that makes an invalid file name will throw an exception (`/` for Windows, but Unix-based systems have more symbols).